src/utils/utils.js

```diff
@@ -37,19 +37,24 @@ export async function createMinimalApp(folderPath) {
 }
 
 export async function createTemplateApp(folderPath, template) {
-  const {
-    stdout
-  } = shelljs.exec(
-    `curl https://api.github.com/repos/sasjs/template_${template}`,
-    { silent: true }
+  const { stdout, stderr, code } = shelljs.exec(
+    `git ls-remote git@github.com:sasjs/template_${template}`,
```

Comment:
Will this approach (git@github) still work even if the user does not have an ssh key?

Reply:
yes, because our template repositories are public. No ssh key is required.